File: self-directed projects/scrabble/guess.py
import random
number = random.randint(0,100)
guess = int(input("guess my number: "))

while(guess != number):
    if(guess < number):
        print("that number is too low!")
    else:
        print("that number is too high!")
    guess = int(input("guess my number: "))

print("congratulations! You've guessed my number!")

File: self-directed projects/scrabble/guess.c
#include <stdio.h>
#include <stdlib.h





:wq

int main(int argc, const char * argv[]){
    int r = rand() % 100;
    char input[10];
    char anewline[5];
    int num = -1;
    while (num != r){
        printf("\n\nguess the number: ");
        fgets(input, sizeof(input), stdin);
        sscanf(input, "%[^\n]%[\n]", input, anewline);
        num = atoi(input);
        if(num > r){
            printf("\nthat number is too high!");
        }
        else if(num < r){
            printf("\nthat number is too low!");
        }
    }
    printf("\n congratulations, you guessed it!");
    return 0;
}
